test/distributed/org/apache/cassandra/distributed/test/repair/AutoRepairSchedulerTest.java

```diff
@@ -112,17 +134,41 @@ public void testScheduler() throws ParseException
 
         // validate that the repair ran on all nodes
         cluster.forEach(i -> i.runOnInstance(() -> {
-            Util.spinAssert("AutoRepair has not yet completed one FULL repair cycle",
+            // Reduce sleeping if repair finishes quickly to speed up test but make it non-zero to provoke some
+            // contention.
+            AutoRepair.SLEEP_IF_REPAIR_FINISHES_QUICKLY = new DurationSpec.IntSecondsBound("1s");
```

Comment:
Noticed the 5 second sleep implicitly added at end of repair cycle slowed down the test quite a bit, setting this to 1s allows the test to complete in ~1m25s